transform.adoc

```diff
@@ -71,5 +71,6 @@ Operations allow further mutation of data, for mutation types that are not curre
 | toCamelCase | string | string | delimiter | The delimiter to split the string on
 | removeHTML | string | string ||
 | convertToFloat64 | string, int, float64 | float64 ||
+| convertToFloat64 | string, int, float64 | int64 ||
```

Comment:
Wrong name here.

Reply:
Good catch, must've snuck into the commit